src/components/FileRepo/statsQueries.tsx

```diff
@@ -0,0 +1,107 @@
+import { Queries } from '../CohortBuilder/QueriesResolver';
+// @ts-ignore
+import filesize from 'filesize';
+
+const formatCount = (numOrString: number | string): string => numOrString.toLocaleString();
+
+const fromSqonValueToVariables = (content: any): any => {
+  if (!content) {
+    return;
+  }
+  return {
+    sqon: content,
+  };
+};
+
+const queryFileStat: string = `
+  query($sqon: JSON) {
+    data: file {
+      hits(filters: $sqon) {
+        total
+      }
+    }
+  }
+`;
+
+const getFileStatHits = (rawData: any): string => formatCount(rawData?.data?.data?.hits?.total);
+
+export const buildStatQueriesForFileHits = (sqonContent: Object): Queries => [
+  {
+    query: queryFileStat,
+    variables: fromSqonValueToVariables(sqonContent),
+    transform: getFileStatHits,
+  },
+];
+
+const queryParticipantStat: string = `
```

Comment:
no need to type, the type infered knows its a string when you assigned a value at the same time as declaring it.

valid for other assignment in this file